trajopt_sco/src/modeling_utils.cpp

```diff
@@ -248,13 +248,14 @@ ConvexConstraints::Ptr ConstraintFromErrFunc::convex(const DblVec& x, Model* mod
 {
   Eigen::VectorXd x_eigen = getVec(x, vars_);
   Eigen::MatrixXd jac = (dfdx_) ? dfdx_->call(x_eigen) : calcForwardNumJac(*f_, x_eigen, epsilon_);
-  ConvexConstraints::Ptr out(new ConvexConstraints(model));
+  auto out = std::make_shared<ConvexConstraints>(model);
   Eigen::VectorXd y = f_->call(x_eigen);
   for (int i = 0; i < jac.rows(); ++i)
   {
     AffExpr aff = affFromValGrad(y[i], x_eigen, jac.row(i), vars_);
     if (coeffs_.size() > 0)
     {
+      /** @todo should not compare floats */
```

Comment:
Nice.

Reply:
Yea, while going through line by line I found this.